e2etests/test/server_valid.js

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var serviceSdk = require('azure-iothub');
+var serviceSas = require('azure-iothub').SharedAccessSignature;
+var provisioningServiceClient = require('azure-iot-provisioning-service').ProvisioningServiceClient;
+var Message = require('azure-iot-common').Message;
+var anHourFromNow = require('azure-iot-common').anHourFromNow;
+var Registry = require('azure-iothub').Registry;
+var NoRetry = require('azure-iot-common').NoRetry;
+var deviceSdk = require('azure-iot-device');
+
+
+var uuid = require('uuid');
+
+var hubConnectionString = process.env.IOTHUB_CONN_STRING_INVALIDCERT;
```

Comment:
see the variable names added in the next commit

Reply:
ok.

---
In reply to: [220400670](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r220400670) [](ancestors = 220400670)